pkgs/development/python-modules/priority/default.nix

```diff
@@ -1,28 +1,31 @@
-{ lib, buildPythonPackage, fetchPypi, pytest, hypothesis }:
+{ lib
+, buildPythonPackage
+, pythonOlder
+, fetchPypi
+, hypothesis
+, pytestCheckHook
+}:
 
 buildPythonPackage rec {
   pname = "priority";
   version = "2.0.0";
+  format = "setuptools";
```

Comment:
Isn't `format = "setuptools";` the default?

Reply:
I think we plan to move away from a default at some point, since more packages are going to be pyproject based.